lib/markdownBuilder.js

```diff
@@ -853,7 +853,7 @@ function build({
 
   console.log('generating markdown');
   return (schemas) => foldl(schemas, {}, (pv, schema) => {
-    const slugger = ghslugger();
+    const slugger = new GhSlugger();
```

Comment:
using new keyword is needed based on `@types/github-slugger` and, in consequence, the case change is required.